pyproject.toml

```diff
@@ -101,7 +101,7 @@ colorhash = ">=1.0.2,<1.3.0"
 jsonschema = ">=3.2,<4.17"
 packaging = ">=20.0,<21.0"
 pytz = ">=2019.1,<2023.0"
-rasa-sdk = "~3.3.0"
+#rasa-sdk = "~3.3.0"
```

Comment:
can we release an alpha for the SDK?

Reply:
As soon as we merge the rasa-sdk PR🤞🏻 @sanchariGr is it ok to release the sdk alpha earlier than scheduled on 28th Nov?

Reply:
Would you prefer it be released as soon as the PR is merged? Sure we can do that 👍

Reply:
Updated to `3.4`: https://github.com/RasaHQ/rasa/pull/11773/commits/94dc6610ddfebd797ff87170b709847a9c57af6d